OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -1,834 +1,1104 @@
+            case CommandType::SetCulling: return "SetCulling";
+            case CommandType::SetPolygonMode: return "SetPolygonMode";
+            default: return "Other";
+        }
+    }
+
+    // ========================================================================
+    // CSV Export
+    // ========================================================================
 
-        return std::string(packet->GetCommandTypeString());
+    std::string CommandPacketDebugger::GenerateExportFilename(const char* extension, u32 frameNumber)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)